examples/server_updating.py

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env python3
-"""Pymodbus asynchronous Server Example.
+"""Pymodbus asynchronous Server with updating subroutine Example.
 
-An example of a multi threaded asynchronous server.
+An example of an asynchronous server with multiple threads and
```

Comment:
Actually it does not have threads, the correct wording is tasks. Threads can be done in both sync and async, but task are solely async.

Reply:
Till open